### ISSUE TITLE
Unnecesary translation `code_bits.__program_indented__`

### ISSUE THREAD

**Issue by oskarissimus** [alexmojaki/futurecoder#443]
why do we have ability to provide this translation, it seems like template sting that is only visible for devs but not for users, so why translate it?
```po
msgid "code_bits.__program_indented__"
msgstr "__program_indented__"
```
its good that it gets caught in tests, but it seems it should be removed

**Comment by alexmojaki** [comment]
Yes, there's a few special terms like this that are generated automatically (by https://github.com/alexmojaki/futurecoder/blob/master/translations/generate_po_file.py) that shouldn't be translated and would ideally be removed, I just didn't bother because it was easy enough to copy the translation.

**Comment by oskarissimus** [comment]
Ok, I'll try to look into this :) thanks for showing starting point :)
